docs/changelog.rst

```diff
@@ -10,18 +10,19 @@ Changes will be added here periodically from the "Suggested changelog entry"
 block in pull request descriptions.
 
 
-
 IN DEVELOPMENT
 --------------
 
 Changes will be summarized here periodically.
 
-Version 2.10.1 (Oct 2?, 2022)
+Version 2.10.1 (Oct 24, 2022)
 -----------------------------
 
+There is some concern about ABI compatibility between 2.10.x (including this release) and previous versions, possibly related to cross-module exception handling. This is still developing and tracked under `#4105 <https://github.com/pybind/pybind11/pull/4105>`_. Please review if this is important to you before upgrading, and report an issue or comment on an issue if there's a problem.
 
-Changes:
+This is the first version to support embedding the newly released Python 3.11.
```

Comment:
I'd insert "fully" here: to fully support embedding

Almost everything works with 2.10.0, with the exception of `PYTHONPATH`.